README.md

```diff
@@ -110,6 +110,50 @@ curl localhost:8080
 # Output: Hello World from a PHP HTTP function!
 ```
 
+## Run your function on Google Cloud Functions
+
+To run your function on Cloud Functions, first you must have the [gcloud SDK][gcloud] installed and [authenticated][gcloud-auth].
+
+Additionally, you need to have a Google Cloud project ID for the
+[Google Cloud Project][gcp-project] you want to use.
+
+> **Note:** PHP support on Cloud Functions is currently in limited alpha.
+> It is not yet suitable for production workloads, and support is best-effort
+> only. Access is currently limited to selected early-access users.
```

Comment:
We should include a link to the alpha signup form if there is one

Reply:
Good idea. I'll ask PM.

Reply:
Added